extensions/bigtable/install_hbase_bigtable.sh

```diff
@@ -78,3 +80,29 @@ echo -e "YARN_OPTS=\"\${YARN_OPTS} -Dyarn.app.mapreduce.am.command-opts=\"${BIGT
 # the hbase classpath.
 echo -e "HBASE_OPTS=\"\${HBASE_OPTS} ${BIGTABLE_BOOT_OPTS}\"" >> "${HBASE_CONF_DIR}/hbase-env.sh"
 
+# Configure Spark
+if [ -d "/home/hadoop/spark-install/" ]; then
+    SPARK_HOME="/home/hadoop/spark-install"
+    export BIGTABLE_CLASSPATH
```

Comment:
This shouldn't be done here.  If we need this, it should be in the install_bigtable.sh file.